modules/33_CDR/portfolio/declarations.gms

```diff
@@ -22,25 +22,25 @@ p33_co2_rem_rate(rlf)                    "carbon removal rate [fraction of annua
 positive variables
 v33_EW_onfield(ttot,all_regi,rlf,rlf)  "amount of ground rock spread on fields in each timestep [Gt]"
 v33_EW_onfield_tot(ttot,all_regi,rlf,rlf)  "total amount of ground rock on fields, for each climate zone and transportation distance [Gt]"
-v33_FEdemand(ttot,all_regi,all_enty,all_enty,all_te)  "FE demand [TWa]"
-vm_ccs_cdr(ttot,all_regi,all_enty,all_enty,all_te,rlf)  "CCS emissions from CDR [GtC / a]"
+v33_FEdemand(ttot,all_regi,all_enty,all_enty,all_te)  "FE demand of each technology [TWa]"
+vm_ccs_cdr(ttot,all_regi,all_enty,all_enty,all_te,rlf)  "total emissions captured through technologies in the CDR module that enter the CCUS chain + captured emissions from associated FE demand [GtC / a]"
 ;
 
 negative variables
-v33_emi(ttot,all_regi,all_te)               "negative emissions from CDR [GtC / a]"
```

Comment:
When you rename a variable, you might have to remove its fixings, levs, and marginals. Right @LaviniaBaumstark?
Cause if you don't do it, someone who for instance runs a PkBudg policy run and uses NPi that was created with `v33_emi` will get an error. Whether that is fine to use an "outdated" ref run, I don't know :) 
Like so: https://github.com/remindmodel/remind/blob/6fc00aaee2c6f66e7fee70f1ae61e363153e53f0/scripts/start/prepare.R#L530C1-L539C6